zebra/zebra_vxlan.c

```diff
@@ -2582,19 +2582,18 @@ void zebra_vxlan_print_specific_rmac_l3vni(struct vty *vty, vni_t l3vni,
 	struct zebra_mac *zrmac = NULL;
 	json_object *json = NULL;
 
+	if (use_json)
+		json = json_object_new_object();
+
 	if (!is_evpn_enabled()) {
-		if (use_json)
-			vty_out(vty, "{}\n");
+		vty_json(vty, json);
```

Comment:
Recently we added `vty_json_empty()` for such cases, can we use it here as well?

Reply:
I have seen `vty_json_empty()` which explicitly creates json object but in many of the case json object was allocated few lines below, I have moved to beginning of the function and use `vty_json()` display empty `{}`.